light_node/src/configuration.rs

```diff
@@ -971,6 +980,12 @@ impl Environment {
                         .collect(),
                 };
 
+                let maindb_backend: TezedgeDatabaseBackendConfiguration = args
+                    .value_of("maindb-backend")
+                    .unwrap_or("rocksdb")
```

Comment:
@mambisi 
here is a default "rocksdb" but then it means that DEFAULT_CONTEXT_KV_STORE_BACKEND is not used